smk/gene_annotation.smk

```diff
@@ -395,12 +389,13 @@ rule gene_annot_eggnog:
         """
         time (
             mkdir out
-            emapper.py --data_dir {params.eggnog_db} -o tmp \
+            emapper.py --data_dir $(dirname {input.eggnog_db}) -o tmp \
                        --no_annot --no_file_comments --report_no_hits --override --output_dir out -m diamond -i {input.faa} --cpu {threads}
             emapper.py --annotate_hits_table out/tmp.emapper.seed_orthologs \
-                       --data_dir {params.eggnog_db} -m no_search --no_file_comments --override -o tmp --output_dir out --cpu {threads} {params.eggnog_inmem}
+                       --data_dir $(dirname {input.eggnog_db}) -m no_search --no_file_comments --override -o tmp --output_dir out --cpu {threads} {params.eggnog_inmem}
             cut -f 1,5,12,13,14,21 out/tmp.emapper.annotations > out/emapper.out
-            echo -e "#Database version\t$(sqlite3 {params.eggnog_db}/eggnog.db 'select * from version;')" > out/eggNOG.tsv
+            echo -e "#Database version\\t$(emapper.py --data_dir $(dirname {input.eggnog_db}) -v | cut
```

Comment:
Now that the sqlite db is not used directly, wouldnt it be better to use the directory as the input and avoid the three `dirname`s?